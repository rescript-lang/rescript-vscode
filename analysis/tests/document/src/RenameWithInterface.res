let x = 2
//  ^ren y
