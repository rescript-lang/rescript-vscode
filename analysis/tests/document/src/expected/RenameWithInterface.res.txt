Rename document/src/RenameWithInterface.res 0:4 y
[
{
  "textDocument": {
  "version": null,
  "uri": "RenameWithInterface.resi"
  },
  "edits": [{
  "range": {"start": {"line": 0, "character": 4}, "end": {"line": 0, "character": 5}},
  "newText": "y"
  }]
  },
{
  "textDocument": {
  "version": null,
  "uri": "RenameWithInterface.res"
  },
  "edits": [{
  "range": {"start": {"line": 0, "character": 4}, "end": {"line": 0, "character": 5}},
  "newText": "y"
  }]
  },
{
  "textDocument": {
  "version": null,
  "uri": "Cross.res"
  },
  "edits": [{
  "range": {"start": {"line": 18, "character": 28}, "end": {"line": 18, "character": 29}},
  "newText": "y"
  }, {
  "range": {"start": {"line": 21, "character": 28}, "end": {"line": 21, "character": 29}},
  "newText": "y"
  }]
  }
]

